src/sugar/submit.ts

```diff
@@ -54,6 +54,7 @@ async function submitSignedTransaction(
   const request: SubmitRequest = {
     command: 'submit',
     tx_blob: signedTxEncoded,
+    fail_hard: isAccountDelete(signedTransaction),
```

Comment:
With this setup, what's the workflow for users who want to `fail_hard`, but aren't submitting an AccountDelete request?

Reply:
They can write a submit `request` themselves